test/torchaudio_unittest/prototype/functional/functional_test_impl.py

```diff
@@ -149,6 +149,149 @@ def test_oscillator_invalid(self):
         with self.assertWarnsRegex(UserWarning, r"above nyquist frequency"):
             F.oscillator_bank(-nyquist * freqs, amps, sample_rate)
 
+    @parameterized.expand(
+        [
+            # Attack (full)
+            param(
+                num_frames=11,
```

Comment:
since `num_frames` is the same for all params, can it be defined in the test method?

Reply:
no, they are same only by chance. they are coupled withe expected values, so it makes more sense to keep them configurable with the reset of parameters.